core/src/surplus.ts

```diff
@@ -0,0 +1,88 @@
+import type { SurplusAuction, SurplusAuctionBase } from './types';
```

Comment:
Can you try updating this branch from `main`? It seems to be outdated and therefore adding whole files that already exist